hlx_statics/blocks/resource-card/resource-card.js

```diff
@@ -1,5 +1,7 @@
 import {
   removeEmptyPTags,
+  decorateWidth,
+  decorateBackgroundColor,
```

Comment:
You added the new decorating functions but you're not using them here should undo?